### ISSUE TITLE
Benchmarks vs reference implementation

### ISSUE THREAD

**Issue by clbarnes** [bodil/meowhash-rs#11]
Do you have any sense of how this implementation stacks up against the reference implementation, speed-wise?

**Comment by bodil** [comment]
Yes, the benchmarks have them at more or less identical performance.
